tests/baselines/reference/declarationEmitQualifiedAliasTypeArgument.types

```diff
@@ -56,17 +56,17 @@ import { create } from "./bbb";
 
 export const fun = create<T>();
 >fun : () => import("bbb").INode<T>
->    : ^^^^^^       ^^^^^  ^^^^^ ^ 
+>    : ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

```

Comment:
The old reuse here seems dubious to me; `create` is imported, so how are we reusing nodes from another file? Similarly to a lot of what changes in this PR.